src/client/stylesheets/search-results.scss

```diff
@@ -0,0 +1,42 @@
+$error-color: #d4351c;
+$tooltip-background-color: #f3f2f1;
+$tooltip-box-shadow-color: rgba(0, 0, 0, 0.3);
+
+.mrn {
+  .error {
+    color: $error-color;
+  }
+
+  .govuk-tag--red.message {
+    text-decoration: underline dotted;
+    text-underline-position: under;
+    cursor: help;
+  }
+
+  .tooltip {
+    visibility: hidden;
+    position: absolute;
+    padding: 10px;
+    width: 300px;
+    z-index: 9999;
+    background-color: $tooltip-background-color;
+    box-shadow: 2px 2px 8px $tooltip-box-shadow-color;
+    display: block;
```

Comment:
The element that the tooltip can be assigned to is changed to a `p` so the `display: block` can be removed